pgspecial/dbcommands.py

```diff
@@ -28,36 +30,36 @@
 
 @special_command("\\l", "\\l[+] [pattern]", "List databases.", aliases=("\\list",))
 def list_databases(cur, pattern, verbose):
-    query = '''SELECT d.datname as "Name",
+    query = SQL('''SELECT d.datname as "Name",
         pg_catalog.pg_get_userbyid(d.datdba) as "Owner",
         pg_catalog.pg_encoding_to_char(d.encoding) as "Encoding",
         d.datcollate as "Collate",
         d.datctype as "Ctype",
-        pg_catalog.array_to_string(d.datacl, E'\n') AS "Access privileges"'''
+        pg_catalog.array_to_string(d.datacl, E'\n') AS "Access privileges"''')
     if verbose:
-        query += ''',
+        query += SQL(''',
             CASE WHEN pg_catalog.has_database_privilege(d.datname, 'CONNECT')
                     THEN pg_catalog.pg_size_pretty(pg_catalog.pg_database_size(d.datname))
                     ELSE 'No Access'
             END as "Size",
             t.spcname as "Tablespace",
-            pg_catalog.shobj_description(d.oid, 'pg_database') as "Description"'''
-    query += """
+            pg_catalog.shobj_description(d.oid, 'pg_database') as "Description"''')
+    query += SQL("""
     FROM pg_catalog.pg_database d
-    """
+    """)
     if verbose:
-        query += """
+        query += SQL("""
         JOIN pg_catalog.pg_tablespace t on d.dattablespace = t.oid
-        """
+        """)
     params = {}
     if pattern:
-        query += """
+        query += SQL("""
         WHERE d.datname ~ %(schema_pattern)s
```

Comment:
Not sure, but I think this should be different now:

`WHERE d.datname ~ {schema_pattern}`

Reply:
Seems correct, as well as using `Literal(schema)` in the `params`.